src/ace/managers/viewport/tilebuffer.c

```diff
@@ -693,7 +693,7 @@ static inline void tileBufferRedrawAllInternal(tTileBufferManager *pManager, UBY
 	UWORD uwMarginedHeight = pManager->uwMarginedHeight;
 	UWORD uwWrapAroundY = MIN(uwEndY, uwMarginedHeight >> ubTileShift);
 	systemSetDmaBit(DMAB_BLITHOG, 1);
-	Disable(); // Disable and clear CPU caches (if any)
+	systemGetBlitterFromOs();
```

Comment:
This isn't really needed, ACE defaults to blitter ownership and only gives it back to OS for file IO. Should've documented those functions and explain this.

If you're paranoid, you can add a check if blitter is owned by ACE using `systemBlitterIsUsed()` and print an error if it's not.